CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Workflow modifications:
 
 AQUA core complete list:
 - Few graphical adjustments in multiple_maps (#2159)
+- Add `string.py` with tests (#2158)
```

Comment:
This is not the correct description for this PR: It should say "Add description for ECmean diagnostic (#2158)". No need to describe files you added etc, the changelog should describe the basic idea of the PR